lib/mayaUsd/fileio/utils/meshWriteUtils.cpp

```diff
@@ -477,4 +478,76 @@ UsdMayaMeshUtil::assignSubDivTagsToUSDPrim(MFnMesh& meshFn,
     }
 }
 
+void
+UsdMayaMeshUtil::writeVertexData(const MFnMesh& meshFn,
+                                 UsdGeomMesh& primSchema,
+                                 const UsdTimeCode& usdTime,
+                                 UsdUtilsSparseValueWriter& valueWriter)
+{
+    MStatus status{MS::kSuccess};
+
+    const uint32_t numVertices = meshFn.numVertices();
+    VtArray<GfVec3f> points(numVertices);
+    const float* pointsData = meshFn.getRawPoints(&status);
+    if(status)
+    {
```

Comment:
It'd be a little bit cleaner to invert this condition and display the error and return if there's a problem. That way we can save an indent level for the code currently inside this block.

Reply:
see ab0a321451a2c0d8be2d4c5237713e9a86e9eb61